src/Shell/Task/PluginTask.php

```diff
@@ -235,7 +237,7 @@ protected function _modifyAutoloader($plugin, $path)
             $cwd = getcwd();
 
             // Windows makes running multiple commands at once hard.
-            chdir(dirname($path));
+            chdir(ROOT);
```

Comment:
It looks to be logically correct to make this change; `dirname($path)` means `dirname('plugins/`)` - it's implicitly assuming the plugins dir is one level below the root. 

However, I'd rather it was:

    $file = $this->_rootComposerFilePath(); # <-
    ...
    chdir(dirname($file));

There's no need to use a constant.